tools/accuracy_checker/accuracy_checker/annotation_converters/criteo_kaggle_dac.py

```diff
@@ -83,6 +90,47 @@ def parse_sparse_features(self):
                 else:
                     ConfigError('Invalid configuration option {}'.format(feat))
 
+    def load_data_file(self):
+        if self.binary:
+            bytes_per_feature = 4
+            tar_fea = 1  # single target
+            den_fea = 13  # 13 dense  features
+            spa_fea = 26  # 26 sparse features
+            tad_fea = tar_fea + den_fea
+            tot_fea = tad_fea + spa_fea
+            self._bytes_per_entry = bytes_per_feature * tot_fea * self.batch
+            self._fea_shape = (self.batch, tot_fea)
+
+            self.count = math.ceil(os.path.getsize(self.src) / self._bytes_per_entry)
```

Comment:
we avoid to use os.path in AC, please use pathlib:
https://docs.python.org/3/library/pathlib.html#pathlib.Path.stat

Reply:
Fixed